packages/core/src/index.ts

```diff
@@ -695,13 +695,37 @@ class Hopr extends EventEmitter {
     }
   }
 
+  /**
+   * @returns a list connected peerIds
+   */
   public getConnectedPeers(): PeerId[] {
     if (!this.networkPeers) {
       return []
     }
     return this.networkPeers.all()
   }
 
+  /**
+   * Takes a look into the indexer.
+   * @returns a list of announced multi addresses
+   */
+  public async getAnnouncedAddresses(): Promise<Multiaddr[]> {
```

Comment:
See my previous comment. If already renaming, this should be called `getAddressesAnnouncedOnChain` (or something similar).